gemini/use-cases/retrieval-augmented-generation/developer_code_chat/developer_code_chat.ipynb

```diff
@@ -0,0 +1,966 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #35.            print("Created dataset {}.{}".format(client.project, dataset.dataset_id))
</pre><p>Use fstrings instead of .format throughout</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/generative-ai/pull/493/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>